selfdrive/car/honda/hondacan.py

```diff
@@ -181,6 +181,9 @@ def create_ui_commands(packer, pcm_speed, hud, car_fingerprint, is_metric, idx,
       'SET_TO_1' : 0x01,
     }
     commands.append(packer.make_can_msg('RADAR_HUD', bus_pt, radar_hud_values, idx))
+    
+    if car_fingerprint in CAR.CIVIC_BOSCH:
```

Comment:
I doubt this will ever be true since `CAR.CIVIC_BOSCH` is string, not a list. Are you sure this change does anything?

Reply:
Without this, the car won't accelerate/brake. The civic_bosch guys can vouch for this one.

Reply:
Ah, my bad. It would be true, but so would a bunch of other subsets. You want either:

`if car_fingerprint in (CAR.CIVIC_BOSCH, )` or `if car_fingerprint == CAR.CIVIC_BOSCH`